ide/dune

```diff
@@ -1,11 +1,36 @@
+; IDE Server
 (ocamllex utf8_convert config_lexer coq_lex)
 
 (library
  (name core)
- (public_name coqide.core)
+ (public_name coqide-server.core)
  (wrapped false)
- (modules (:standard \ idetop coqide_main))
- (libraries threads str lablgtk2.sourceview2 coq.lib coqide.protocol))
+ (modules document)
+ (libraries coq.lib))
+
+(executable
+ (name fake_ide)
+ (public_name fake_ide)
```

Comment:
If this is installed, then I guess we should follow the convention of prefixing it with `coq`. Note that it isn't installed in the make-based build (at least not in `/bin`).

Reply:
Good point, I was indeed not sure whether to install it or not. If we install it, it'll live in `PATH` which is convenient, if we don't install it maybe indeed should be just moved to `test-suite/ide` and made private to it?

Reply:
Yes, it would make more sense IMHO to be made private to test-suite.

Reply:
Done then, it is easy to change anyways.